### ISSUE TITLE
Ability to have custom time bar svg

### ISSUE THREAD

**Issue by khanzzirfan** [xzdarcy/react-timeline-editor#19]
Hello Team,
is there Ability to customize the timeline head with separate svg icon?

**Comment by xzdarcy** [comment]
Does [this example](http://localhost:8000/editor-demo/editor-scroll-sync) meet the needs? Any content can be defined on the left side of the timeline and synchronized with the timeline scrolling. 
I think it may not be very appropriate to define operable icons in the row because it will follow the horizontal scrolling.

**Comment by khanzzirfan** [comment]
hey, i think you provided the Localhost url 8080.. which I cannot view the example. Please see what your example URL. 

Also the ask is,  see the image. it should have the timeline head as rounded icon customizable.

![image](https://user-images.githubusercontent.com/8625166/210696069-9d3ce330-380e-41c6-9ef3-4059c3411660.png)

**Comment by xzdarcy** [comment]
Sorry, i just updated the example link below. Add do you mean to customize the style here?
![image](https://user-images.githubusercontent.com/43289714/210697696-c419d12a-24ff-4fd5-8e10-b9af40f0f08a.png)

**Comment by khanzzirfan** [comment]
Yes, thats right. Want to customize the head of the scrubber bar
